hg-fast-export.py

```diff
@@ -183,8 +185,27 @@ def export_file_contents(ctx,manifest,files,hgtags,encoding='',plugins={}):
         b'Ignoring file %s which cannot be tracked by git\n' % filename
       )
       continue
-    file_ctx=ctx.filectx(file)
-    d=file_ctx.data()
+    if filename[:6] == b'.hglf/':
+      filename = filename[6:]
+      sys.stderr.write("Detected large file named %s\n" % filename.decode())
+      lf_file_ctx=ctx.filectx(file)
+      lf_hash=lf_file_ctx.data().strip(b'\n')
+      sys.stderr.write("Detected large file hash %s\n" % lf_hash.decode())
+      #should detect where the large files are located
+      file_with_data = lfutil.findfile(ctx.repo(), lf_hash)
+      if file_with_data is None:
+        # Autodownloading from the mercurial repository would be an issue as there is a good chance that we may
+        # need to input some username and password. This will surely break fast-export as there will be
+        # some unexpected output.
+        sys.stderr.write("Large file wasn't found in local cache.\n")
+        sys.stderr.write("Please clone with --all-largefiles\n")
+        sys.stderr.write("or pull all large files with 'hg lfpull --rev \"all()\"'\n")
+        sys.exit(3) # closing in the middle of import will revert everything to the last checkpoint
+      with open(os.path.normpath(file_with_data), 'rb') as file_with_data_handle:
+        d=file_with_data_handle.read()
```

Comment:
Please line-break this region to fit in 80 columns (I know we're sinning on this in other places, but I don't want to add new code doing it).